python/server/qgsserviceregistry.sip

```diff
@@ -0,0 +1,83 @@
+/***************************************************************************
+                          qgsserviceregistry.h
+
+  Class defining the service manager for QGIS server services.
+  -------------------
+  begin                : 2016-12-05
+  copyright            : (C) 2016 by David Marteau
+  email                : david dot marteau at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+/**
+ * \ingroup server
+ * QgsServiceRegistry
+ * Class defining the cegistry manager for QGIS server services
+ *
+ * This class provides methods for registering and retrieving 
+ * services. Note that the regstiry does not hord ownership of
+ * registered service but vill call the 'release' method on cleanup)
+ *
+ */
+class QgsServiceRegistry
```

Comment:
It would be also useful to have a method to get a list of registered services...